### ISSUE TITLE
Greeting will not play

### ISSUE THREAD

**Issue by brynlcdjs** [playfultechnology/audio-guestbook#129]
Hello I have tried so many different things at this point to get my guestbooks (I have 3, 2 of them have this issue 1 somehow works fine) to work such as replacing the greeting and guest folders, making sure the greeting file is .wav, and even formatting my micro SD cards with zero luck. No matter what I do, when I pick up the handset I get a beep then it starts recording. I need to get these back to the point of playing the pre recorded greeting and am really at the end of my rope trying to figure anything out. Any advice at all would be massively appreciated.

**Comment by Teflon843** [comment]
Hello, did you ever resolve your issue?

**Comment by brynlcdjs** [comment]
Yes I finally heard back from the company who we purchased from, they sent the original file which is working. I guess it needed to be specifically labeled GREETING.mp3 even though I tried changing another one to that and it didn't work. So I am not 100% sure what the exact issue was but it is working now.
